steps/setup_backend_docker_generic.yml

```diff
@@ -3,16 +3,33 @@ hints:
 - Pssst - you can run `docker-compose up` from the root directory
 - run `docker exec -it <container-name> sh` to access a container from the terminal.
 startFlow:
-  do:
-  - actionId: bot_message
-    params:
-      person: head-of-rd
-      messages:
-      - text: If Docker is working correctly, the backend should be running and able
-          to connect to your local database.
-        delay: 2000
-      - text: Let's test this by **pointing your browser** to http://localhost:3000/api/ping
-        delay: 1000
+  if:
+    conditions:
+    - conditionId: user_has_feature_flag
+      params:
+        featureFlag: 'codespaces'
+    then:
+      do:
+      - actionId: bot_message
+        params:
+          person: head-of-rd
+          messages:
+          - text: If docker-compose is working correctly and finished loading up, the backend should be running and able
+              to connect to your local database.
```

Comment:
This message is for users running with codespace?
docker-compose is not running locally right?

Reply:
oh nice catch! will change, thanks :)